blog/_posts/2016-05-27-scala-center-advisory-board.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: blog
+post-type: blog
+by: Jon Pretty
+title: The Scala Center Advisory Board
+---
+
+# The Scala Center Advisory Board
+
+Part of the role of the Scala Center is to run the Advisory Board, a quarterly
+forum for our sponsors to discuss, guide and challenge the direction of the
+Scala Center and how it influences the Scala community and ecosystem.
+
+It was never intentional, but it's become a bit of a joke at EPFL that the
+"Scala Center Advisory Board" was born with an unfortunate acronym! So, when I
+chaired our inaugral meeting in New York two weeks ago, one of the first points
+I made to the attendees was that we have to exceed our acronym and earn the
+community's trust and respect, ensuring we provide a worthwhile and beneficial
+service to all users of Scala.
+
+This respect is not something to be assumed or taken for granted, though we
+have received a lot of support already from many Scala users. The Advisory
+Board's legitimacy as a "guiding light" to the Scala Community is quite rightly
+something we should expect to be challenged on, so it is paramount that we do
+the best with the power we have been entrusted with.
+
+So, what is the Advisory Board?
+
+## About the Advisory Board
+
+The Advisory Board is a separate body from the Scala Center, much as many
+governments have separate legislative and executive branches: the Advisory
+Board makes recommendations to the Scala Center on the work we should do, but
+it's the Scala Center's job to execute those recommendations.
+
+It currently has seven voting members: representatives from each of our six
+sponsors, plus Bill Venners, the community representative. Additionally the
+Executive Director of the Scala Center, Heather Miller, sits on the board to
+report on the Scala Center's activities, and provide advice on the feasibility
+of the proposals under consideration, and Martin Odersky is the technical
+advisor to the board.
+
+The board will meet four times a year, usually via a conference call to keep
+costs down, but when there's an opportunity (hopefully at least once a year) we
+will hold physical meetings. We organized our first meeting to take place at
+Scala Days in New York, as most of the sponsors were attending. (The sponsors
```

Comment:
Just to keep with terminology. We normally refer to them as advisory board members, no?